packages/akashic-cli-export/src/zip/convert.ts

```diff
@@ -143,7 +145,11 @@ export function convertGame(param: ConvertGameParameterObject): Promise<void> {
 				operationFiles.forEach(file => {
 					const realPath = path.join(path.dirname(scriptPath), file).replace(`${param.source}/`, "");
 					if (files.indexOf(realPath) !== -1) {
+						// TODO: 操作プラグインによって preserve するファイルは bundle から除外する必要がある
 						preservingFilePathSet.add(realPath);
+						if (bundledFilePaths.indexOf(realPath) !== -1) {
+							console.warn(`${realPath} is duplicated`);
```

Comment:
エラーメッセージの類は、できるだけ「どうすればいいのか」をセットで提示したいです。まず滅多に通らないパスですし、長めに書いてこんな感じでどうでしょうか。

`${realPath} IS DUPLICATED: the file is required from both the game and an operation plugin. The current akashic export command duplicates the file. This may breaks the code, especially using the instanceof operator. Try --no-bundle option if you have a trouble.`

Reply:
あ、、どうすべきかの記述を全くしてませんでした。。ありがとうございます、ここはそのように文言を修正しておきます